src/components/ProfilePage/Certificates.jsx

```diff
@@ -74,7 +75,11 @@ class Certificates extends React.Component {
           <CardBody className="d-flex flex-column">
             <CardTitle>
               <p className="small mb-0">
-                {intl.formatMessage(messages[`profile.certificates.types.${certificateType}`])}
+                {intl.formatMessage(get(
+                  messages,
+                  `profile.certificates.types.${certificateType}`,
+                  messages['profile.certificates.types.verified'],
```

Comment:
I'm not sure it is appropriate to default to verified here. Do we know what the certificate type was that was triggering the error?

Reply:
No I didn't check :/ – but we need a default anyway. Maybe we just add one that says 'Certificate'? Or nothing?

Reply:
Agreed.
1. We should fix the actual problem by adding a certificate type for whatever caused the error.
2. Tagging @marcotuts in case he wants to weigh-in on the default in case this shows up in the future.

Reply:
It looks like some of us have certificates with a type of "audit" on stage which I think is what was causing the issue. @abutterworth Yeah, let's create a separate default certificate type translation id for unknown types. "Certificate" makes sense to me as the default translation string.